clients/admin-ui/cypress/fixtures/connections.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
this already exists here: https://github.com/ethyca/fides/blob/main/clients/admin-ui/cypress/fixtures/connectors/list.json

can we use that one instead?